amxmodx/file.cpp

```diff
@@ -96,12 +96,14 @@ static cell AMX_NATIVE_CALL read_file(AMX *amx, cell *params)
 	{
 		length = strlen(buffer);
 
-		if (buffer[length - 1] == '\n')
-			buffer[--length] = '\0';
-
-		if (buffer[length - 1] == '\r')
-			buffer[--length] = '\0';
-
+        if (length > 0)
```

Comment:
Indentation is wrong here - looks like your editor is not inserting hard tabs.

Reply:
Yes sorry